tests/bagatom/test_wrapAtom.py

```diff
@@ -0,0 +1,202 @@
+from lxml import etree
+from datetime import datetime
+
+from codalib import bagatom
+
+xml = """<note>
+            <to>Tove</to>
+            <from>Jani</from>
+            <heading>Reminder</heading>
+            <body>Don't forget me this weekend!</body>
+        </note>
+"""
+
+
+def test_return_type():
+    """
+    Verify the return type of wrapAtom is an instance of etree._Element.
+    """
+    root = etree.fromstring(xml)
+    atom = bagatom.wrapAtom(root, "934023", "test")
+    assert isinstance(atom, etree._Element)
+
+
+def test_has_date():
+    """
+    Check the returned tree has an updated element by default.
+    """
+    root = etree.fromstring(xml)
+    atom = bagatom.wrapAtom(root, "934023", "test")
+
+    updated = atom.xpath(
+        '/a:entry/a:updated',
+        namespaces={'a': bagatom.ATOM_NAMESPACE}
+    )
+    assert len(updated) == 1
+
+
+def test_uses_date():
+    """
+    Check the returned tree uses the `updated` kwarg text for the
+    updated element.
+    """
+    updated = datetime(2012, 12, 12)
+
+    root = etree.fromstring(xml)
+    atom = bagatom.wrapAtom(root, "934023", "test", updated=updated)
+
+    elements = atom.xpath(
+        '/a:entry/a:updated',
+        namespaces={'a': bagatom.ATOM_NAMESPACE}
+    )
+    assert len(elements) == 1
+
+    date = datetime.strptime(elements[0].text, bagatom.TIME_FORMAT_STRING)
+    assert date == updated
+
+
+def test_has_title():
+    """
+    Check that the title element is present.
+    """
+    title = "Test Title"
```

Comment:
Looks like each instance of the atom var, as well as some of the others, need to have their double quotes changed.